pkgs/by-name/wi/windterm/package.nix

```diff
@@ -0,0 +1,95 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  fontconfig,
+  freetype,
+  libGL,
+  glib,
+  alsa-lib,
+  pulseaudio,
+  qt5,
+  xorg,
+  gtk3,
+  atk,
+  pango,
+  gdk-pixbuf,
+  cairo,
+  gst_all_1,
+  makeWrapper,
+  profiles ? {
+    path = "~";
+  },
+}:
+let
+  version = "2.6.1";
+  src = fetchurl {
+    url = "https://github.com/kingToolbox/WindTerm/releases/download/2.6.0/WindTerm_${version}_Linux_Portable_x86_64.tar.gz";
+    hash = "sha256-JwTsfUkESl2vUx48TanKYAOVWw6q4xGY+i0PrN9GfpA=";
+  };
+in
+stdenv.mkDerivation {
+  inherit version src;
+
+  pname = "windterm";
+
+  dontBuild = true;
+
+  nativeBuildInputs = [
+    makeWrapper
+    qt5.wrapQtAppsHook
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    fontconfig
+    freetype
+    libGL
+    glib
+    alsa-lib
+    pulseaudio
+    gst_all_1.gst-plugins-base
+    xorg.libXtst
+    gtk3
+    atk
+    pango
+    gdk-pixbuf
+    cairo
+    stdenv.cc.cc.lib
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/windterm/ $out/bin/ $out/share/applications/ $out/share/pixmaps/
+    cp -r ./* $out/windterm/
+    chmod 755 $out/windterm/
+    cat > $out/windterm/profiles.config <<EOF
+    ${builtins.toJSON profiles}
+    EOF
+    find $out/windterm -type f -exec chmod 644 {} +
+    find $out/windterm -type d -exec chmod 755 {} +
+    chmod +x $out/windterm/WindTerm
```

Comment:
These files should not be copied over as a whole? Are there anything preventing us from copying each file to standard location directly?

Reply:
Does the executable require a relative path structure within `$out/windterm` to work properly? If not, we can use commands such as `install -Dm755` to directly put the files in standard locations such as `$out/bin`, `$out/lib` and so on. Also the `./` contains source code so I'm not sure about the intention of copying them over to `$out` as a whole.